src/test/java/edu/usf/cutr/tba/test/TravelBehaviorUtilsTest.java

```diff
@@ -504,4 +505,51 @@ public  void testGetDateAndTimeFromMillis() {
         assertEquals("2019-08-10T07:09:52Z", TravelBehaviorUtils.getDateAndTimeFromMillis(testDate3));
         assertEquals("2019-08-11T03:36:22Z", TravelBehaviorUtils.getDateAndTimeFromMillis(testDate4));
     }
+
+    /**
+     * Given different list of LocationInfo objects, verify that the getLocationInfo method
+     * return an objects that matches the location provider or the null object if not match
+     * provider is found.
+     */
+    @Test
+    public void testGetLocationInfo() {
+        List<TravelBehaviorInfo.LocationInfo> locationInfoEmptyList = null;
+        List<TravelBehaviorInfo.LocationInfo> locationInfoFullList = null;
+        List<TravelBehaviorInfo.LocationInfo> locationInfoPartialList = null;
+
+        TravelBehaviorInfo.LocationInfo locationInfoFused = new TravelBehaviorInfo.LocationInfo();
+        locationInfoFused.provider = "fused";
```

Comment:
For readability I'd suggest refactoring this so that the variable names are just `fused` instead of `locationInfoFused`, etc

Reply:
The change will be reflected in the next PR.